lib/server.js

```diff
@@ -30,11 +30,13 @@ export async function startServer(options, extraArgs = "") {
 
   log("Waiting for server to start...");
 
+  let timeout = parseNumber(options.timeout, { fallback: 120_000 });
```

Comment:
You don't need a fallback here because commander will always give you a default value when the default is specified. And if the parse fails we should just throw an error

Reply:
sg